lib/graphql-hive/client.rb

```diff
@@ -20,23 +20,35 @@ def send(path, body, _log_type)
             path: path
           )
 
+        http = setup_http(uri)
+        request = build_request(uri, path, body) 
+        response = http.request(request)
+
+        @options[:logger].debug(response.inspect)
+        @options[:logger].debug(response.body.inspect)
+      rescue StandardError => e
+        @options[:logger].fatal("Failed to send data: #{e}")
+      end
```

Comment:
I think having a logger is OK here, since it's an error, but we should re-raise the issue to the clients.
```suggestion
      rescue StandardError => e
        @options[:logger].fatal("Failed to send data: #{e}")
        raise e
      end
```